### PR TITLE
fixes lxc-webpanel/LXC-Web-Panel/issues#102

### PR DESCRIPTION
**Pull request by ceremcem**
fixes lxc-webpanel/LXC-Web-Panel/issues/102

### CODE REVIEW

File: lwp/__init__.py [hunk 1]
```diff
@@ -71,6 +71,8 @@ class CalledProcessError(Exception):
 
 def FakeSection(fp):
     content = u"[DEFAULT]\n%s" % fp.read()
+    print "CONTENT: ", content
```
Comment (ElieDeloumeau) [inline lwp/__init__.py]: Can you remove 'print' pls?

Comment (ceremcem) [inline lwp/__init__.py]: Oh, right, I forgot that

**Comment by ElieDeloumeau** [comment]
Thanks ;)

**Comment by ceremcem** [comment]
You welcome :)

**Pull request merged**